### PR TITLE
Remove gist:Room. Fixes #102

### LINKED ISSUE #102: Problems with gist:Room
The term specification does not define a "room" in the ordinary sense: e.g., elevators, staircases, hallways, and closets are direct parts of buildings but not rooms in the ordinary sense; and they can all have IDs. Also, a room in the ordinary sense can be a direct part of a wing, where the wing is a direct part of a building. In this case the room is a part of the building but not a direct part. According to the gist definition of Room, the wing is a Room and the room within a wing is not a Room. We could rename the class, but the usefulness of a class that includes elevators, staircases, hallways, wings, and some rooms but not others, is dubious; this is confirmed by the difficulty of finding an accurate name for the class - i.e., it's not a real concept. I suggest modifying the definition of gist:Room to be a semantically and coherent reflection of the concept of "room."

### PR DESCRIPTION
**Pull request by rjyounes**
Since this is for major release 10.0.0, the class is removed rather than deprecated.

### CODE REVIEW

File: docs/ReleaseNotes.md [hunk 1]
```diff
@@ -1,23 +1,36 @@
...
+  - `giver` and `getter`
+    - Renamed to `hasGiver` and `hasGetter`
+    - The newly named versions are no longer subproperties of `hasParty`
+    - Deprecated `giver` and `getter`
+  - New property: `hasParticipant`
+    - No domain or range
+    - Has subproperties: `hasGiver`, `hasGetter`, `hasParty`, `fromAgent` and `toAgent`
+  - Added a `skos:scopeNote` to `fromAgent`
+  - Added a `skos:example` to `hasParty`
+  - Updated `skos:definition`s for `toAgent` and `fromAgent`
```
Comment (sa-bpelakh) [inline docs/ReleaseNotes.md:33]: Please revert this change. It _seems_ like the whitespace is extraneous, but when converted to HTML (via build), the old indentation gives the desired result:
image
but without the spaces, the nested lists don't render properly:
image

File: docs/ChangeAndReleaseManagement.md [hunk 2]
```diff
@@ -43,7 +43,7 @@ Additional notes:
 
 - Correction of an error, even if not backward-compatible, does not require a major release. The expectation is that users will not have implemented against an obvious error. This would be a patch.
 - When a local name is altered, the original term is deprecated to make it a minor rather than major change. The deprecated term receives an `owl:equivalentClass` or `owl:equivalentProperty` assertion to the new term. Deprecated terms may be removed in a future major release. Deprecated terms reside in the `gistDeprecated.ttl` file. If a user wants to use a deprecated term, he/she should import this file into his/her ontology, which in turn imports `gistCore.ttl` and thus all of gist.
-- Major changes should have a significant impact aside from technically modifying inferencing if this is low-impact. E.g.,changing an equivalent class axiom to a union class to a subclass axiom in order to allow new subclasses to be defined. 
+- Major changes should have a significant impact aside from technically modifying inferencing if this is low-impact. E.g.,when an equivalent class axiom to a union class is changed to a subclass axiom in order to allow new subclasses to be defined.
```
Comment (sa-bpelakh) [inline docs/ChangeAndReleaseManagement.md:46]:
```suggestion
- Major changes should have a significant impact aside from technically modifying inferencing if this is low-impact. E.g., when an equivalent class axiom to a union class is changed to a subclass axiom in order to allow new subclasses to be defined.
```

**Review by sa-bpelakh — CHANGES_REQUESTED** [review]
Actual changes look fine, minor issue w/ documentation formatting.

**Review by sa-bpelakh — APPROVED** [review]
You are going to have to educate me about this Markdown lint tool you are using 🤔

File: docs/ReleaseNotes.md (hunk 1 above)
Comment (rjyounes) [reply docs/ReleaseNotes.md]: I've fixed this. Can you check if there are similar problems with other nested lists in the file? We have a two-level nested list in 9.5.0 notes. This is the only three-level one we have.
